packages/cli/src/commands/apps.ts

```diff
@@ -35,27 +35,27 @@ export default class AppsCommand extends APICommand {
 	}]
 
 	async run(): Promise<void> {
-		const { args, argv, flags } = await this.parse(AppsCommand)
-		await super.setup(args, argv, flags)
-
 		const config = {
 			primaryKeyName: 'appId',
 			sortKeyName: 'displayName',
 			tableFieldDefinitions: tableFieldDefinitions,
 			listTableFieldDefinitions: ['displayName', 'appType', 'appId'],
 		}
 
-		if (flags.verbose) {
+		if (this.flags.verbose) {
 			config.listTableFieldDefinitions.push('ARN/URL')
 		}
 
 		const listApps = async (): Promise<App[]> => {
 			const appListOptions: AppListOptions = {}
-			if (flags.type) {
-				appListOptions.appType = AppType[flags.type as keyof typeof AppType]
+			if (this.flags.type) {
+				appListOptions.appType = AppType[this.flags.type as keyof typeof AppType]
 			}
-			if (flags.classification) {
-				appListOptions.classification = flags.classification.map(it => AppClassification[it as keyof typeof AppClassification])
+
+			if (this.flags.classification) {
+				// tradeoff for not parsing in here is that TS cannot infer typeof this.constructor.flags
+				const classifications = this.flags.classification as string[]
+				appListOptions.classification = classifications.map(classification => AppClassification[classification as keyof typeof AppClassification])
```

Comment:
I tried a number of things both here and in out base command, but couldn't get flags to infer the static flags types without explicit casting.